tests/bin/agent/status.test.ts

```diff
@@ -54,22 +53,26 @@ describe('status', () => {
           '--verbose',
         ],
         {
-          PK_NODE_PATH: path.join(dataDir, 'polykey'),
-          PK_PASSWORD: password,
-          PK_ROOT_KEY: globalRootKeyPems[0],
+          env: {
+            PK_NODE_PATH: path.join(dataDir, 'polykey'),
+            PK_PASSWORD: password,
+            PK_ROOT_KEY: globalRootKeyPems[0],
+          },
+          cwd: dataDir,
         },
-        dataDir,
         logger,
       );
       await status.waitFor('STARTING');
       let exitCode, stdout;
-      ({ exitCode, stdout } = await execUtils.pkStdio(
+      ({ exitCode, stdout } = await testUtils.pkExec(
```

Comment:
So previously @tegefaulkes changed `pkStdio` to be using `execFile` right?

What we are doing here is:

1. Reverting `pkStdio` to be how it was before, calling the functions and having mocks.
2. The tests that were using `pkStdio` and didn't have problems with the lack of mocking is now using `pkExec`.
3. So `pkStdio` is still preserved for tests that require mocking.

Now I'm a bit confused, how did the tests that required mocking continue to work when `pkStdio` was swapped out to calling `execFile`?

Reply:
I don't think `execFile` was ever used? `pkStdio` was only changed to have a `pkStdioTarget` option, which called `child_process.spawn`. The change here was to remove `pkStdioTarget` and anywhere that `pkStdio` was being used for a docker integration test I changed it to use `pkExec` instead. None of the tests that use mocking were ever affected and continue to use `pkStdio` with mocking as they always have.